Classes/PostProcessor/PublishStatusPostProcessor.php

```diff
@@ -32,8 +32,20 @@ public function process($response): array
     {
         $response = (array)$response;
         foreach ($response as $key => $item) {
+            // Skip _root entry
+            if ($key === '_root') {
+                continue;
+            }
+
+            // Check if leistung is a leistung or a zustaendigkeit with a leistung as array item
+            if (array_key_exists('leistung', $item) && is_array($item['leistung'])) {
+                $leistung = $item['leistung'];
+            } else {
+                $leistung = $item;
+            }
+
             // Remove non published items from array
-            if (!array_key_exists('publishStatus', $item) || $item['publishStatus'] !== 'DONE') {
+            if (!array_key_exists('publishStatus', $leistung) || $leistung['publishStatus'] !== 'DONE') {
```

Comment:
Was hälst davon um `$leistung['publishStatus']` ein strtoupper zu machen. Einfach, um in Zukunft safe zu sein.

Reply:
Da ich dieser API alles zutraue, halte ich das für sinnvoll 👍.